docs/features/creating_container.md

```diff
@@ -11,7 +11,15 @@ up with Testcontainers and integrate into your tests:
 
 `testcontainers.GenericContainer` defines the container that should be run, similar to the `docker run` command.
 
-The following test creates an NGINX container and validates that it returns 200 for the status code:
+The following test creates an NGINX container on both the `bridge` (docker default
+network) and the `foo` network and validates that it returns 200 for the status code.
+
+It also demonstrates how to use `CleanupContainer` ensures that nginx container
+is removed when the test ends even if the underlying `GenericContainer` errored
+as well as the `CleanupNetwork` which does the same for networks.
+
+The alternatives for these outside of tests as a `defer` are `TerminateContainer`
+and `Network.Remove` which can be seen in the examples.
```

Comment:
Cool! I'll also add a follow-up PR to have a dedicated section in the docs for removing containers in tests and testable examples.

Thanks for this!